From 98acd2853bf395ca3506d9823eb60c52b6010820 Mon Sep 17 00:00:00 2001
From: Szabolcs Nagy <nsz@port70.net>
Date: Wed, 22 Aug 2018 17:56:59 +0000
Subject: [PATCH 8/8] add NT_VMCOREDD to elf.h from linux v4.18

used for device driver dump in /proc/vmcore
new in linux commit 2724273e8fd00b512596a77ee063f49b25f36507
---
 include/elf.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/elf.h b/include/elf.h
index 4a8843c7..54f41a10 100644
--- a/include/elf.h
+++ b/include/elf.h
@@ -685,6 +685,7 @@ typedef struct {
 #define NT_METAG_RPIPE	0x501
 #define NT_METAG_TLS	0x502
 #define NT_ARC_V2	0x600
+#define NT_VMCOREDD	0x700
 #define NT_VERSION	1
 
 
-- 
2.17.1

